assets/dns/daemonset.yaml

```diff
@@ -15,6 +15,7 @@ spec:
         openshift-app: dns
     spec:
       serviceAccountName: dns
+      priorityClassName: system-cluster-critical
```

Comment:
Should this be `system-node-critical`? Without the operator, the cluster is broken; without the local CoreDNS instance, the node is broken.

Reply:
Hmm, could be, especially now that we're responsible for image registry connectivity via /etc/hosts on the node. @aveshagarwal, thoughts?

Reply:
Yes, I think this should be `system-node-critical` (should never be evicted from the node).

Reply:
+1 for system-node-critical for dns pods.

Reply:
Updated to use system-node-critical